app/cli.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+"""
+Command-line interface for running the Script Runner web application.
+"""
+
+
+import os
+
+import click
+
+
+@click.command()
+@click.option(
+    "--config",
+    envvar="CONFIG_FILE_PATH",
+    help="Path to the configuration file.",
+    required=True,
+    type=click.Path(exists=True, file_okay=True, dir_okay=False, readable=True),
+)
+@click.option(
+    "--host",
+    required=True,
+    help="Host to bind to.",
+)
+@click.option(
+    "--port",
+    required=True,
+    help="Port to bind to.",
+    type=int,
+)
```

Comment:
we can apply the 127.0.0.1:5000 defaults in the CLI here, we probably won't need to change that